package.json

```diff
@@ -198,6 +198,7 @@
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
     "timers-browserify": "^2.0.12",
+    "trim": "^0.0.3",
```

Comment:
I don't think we need to specify this in `devDependencies` is we aren't directly using it. Keep in `resolutions`